modules/virt-creating-vm-cli.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-cli_{context}"]
+= Using the CLI to create a virtual machine
+
+You can create a virtual machine from a `virtualMachine` manifest.
+
+.Procedure
+
+. Edit the `VirtualMachine` manifest, according to the following example for a RHEL virtual machine, and save it as `vm.yaml`:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    app: <vm-name>
```

Comment:
throughout the YAML: s/`<vm-name>`/`<vm_name>`

Also, consider adding a callout here to make it clear that if someone _does_ use this manifest, they have to at least change the values that are in `< >`.

Reply:
I think that would be too obvious to require a callout. But I guess it does not hurt to add a callout.